InfrastructureManager/agents/vsphere_agent.py

```diff
@@ -0,0 +1,111 @@
+"""
+[TESTING]
+Agent class for vSphere.
+The agent need pysphere 0.1.8 you can install from pip or from esayinstall tools
+
+You need setup your vCenter:
+    you need create a AppScale template with VMWare tools installed and to be
+    powered on.
+      you need create a head node in vSphere
```

Comment:
Capitalize 'you' and format this docstring better. See other files as an example.